### PR TITLE
Refacto/bootstrapvue+prestakit

### PR DESCRIPTION
**Pull request by v4lux**

### CODE REVIEW

File: _dev/src/pages/Help.vue [hunk 1]
```diff
@@ -18,11 +18,13 @@
  *-->
 <template>
   <div>
-    <div class="row">
-      <div class="container">
-        <HelpPanel module-key="bab535c26e031e9d224c0d128e063283" ps-version="1.6" iso-code="en" />
-      </div>
-    </div>
+    <b-container>
+      <HelpPanel
```
Comment (Matt75) [inline _dev/src/pages/Help.vue]: Params should be dynamic

File: _dev/src/components/panel/active-payment.vue [hunk 2]
```diff
@@ -182,7 +217,7 @@
     display: none;
   }
   .payment-method-container {
-    max-width: 500px;
+    /* max-width: 500px; */
```
Comment (Quetzacoalt91) [inline _dev/src/components/panel/active-payment.vue]: To be removed

File: _dev/src/components/panel/active-payment.vue [hunk 3]
```diff
@@ -198,7 +233,7 @@
     background-color: #fff;
     border: 1px solid #dddddd;
     border-radius: 3px;
-    max-width: 475px;
+    /* max-width: 475px; */
```
Comment (Quetzacoalt91) [inline _dev/src/components/panel/active-payment.vue]: To be removed

File: _dev/src/components/panel/help.vue [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+              <ul class="mt-3">
+                <li>{{ $t('panel.help.tip1') }}</li>
+                <li>{{ $t('panel.help.tip2') }}</li>
+                <li>{{ $t('panel.help.tip3') }}</li>
+                <li>{{ $t('panel.help.tip4') }}</li>
+              </ul>
+            </div>
+          </div>
+          <div class="faq">
+            <h1>FAQ</h1>
```
Comment (Quetzacoalt91) [inline _dev/src/components/panel/help.vue]: Missing translation

File: _dev/src/components/block/dispute.vue [hunk 5]
```diff
@@ -17,24 +17,27 @@
...
-      </a>
-    </div>
-  </div>
+  <b-card no-body>
+    <b-card-body class="py-2">
+      <b-row>
+        <b-col>
+          <b-button variant="text" size="lg">
+            <i class="material-icons">school</i>
+            0 {{ $t('block.dispute.pendingDispute') }}
```
Comment (Quetzacoalt91) [inline _dev/src/components/block/dispute.vue]: `0` should replace a placeholder. Following the doc (https://kazupon.github.io/vue-i18n/guide/formatting.html#named-formatting), we should see something like

```suggestion
            {{ $t('block.dispute.pendingDispute', { disputeCount: 0}) }}
```

With block.dispute.pendingDispute updated as `{disputeCount} pending dispute(s)`

**Pull request merged**